zerver/lib/actions.py

```diff
@@ -231,24 +231,28 @@ def send_signup_message(sender, signups_stream, user_profile,
         internal_blurb = " "
 
     user_count = realm_user_count(user_profile.realm)
-    notifications_stream = user_profile.realm.get_notifications_stream()
+    signup_notifications_stream = user_profile.realm.get_signup_notifications_stream()
     # Send notification to realm notifications stream if it exists
     # Don't send notification for the first user in a realm
-    if notifications_stream is not None and user_count > 1:
+    if signup_notifications_stream is not None and user_count > 1:
         internal_send_message(
             user_profile.realm,
             sender,
             "stream",
-            notifications_stream.name,
-            "New users", "%s just signed up for Zulip. Say hello!" % (
-                user_profile.full_name,)
+            signup_notifications_stream.name,
+            "New users",
```

Comment:
maybe "signups", as the topic?
Also, changing my mind a bit about the content, how about: "%s (%s) just signed up for Zulip. (total: %s)"
The issue is that the email address gets highlighted as a link, but we should just fix that in css. Also removes the bold and the `!`.